### PR TITLE
fix(autofocus): check for truthy attribute

### PR DESCRIPTION
**Pull request by brianchance**
For #1951

### CODE REVIEW

File: src/uiSelectDirective.js [hunk 1]
```diff
@@ -161,7 +161,7 @@ uis.directive('uiSelect',
         });
 
         //Automatically gets focus when loaded
-        if (angular.isDefined(attrs.autofocus)){
+        if (angular.isDefined(attrs.autofocus) && $parse(attrs.autofocus)()) {
```
Comment (Jefiozie) [inline src/uiSelectDirective.js]: Isn’t attrs.$observe not better to use instead of parse?

Comment (brianchance) [reply src/uiSelectDirective.js]: It does not need to $observe for changes, just the initial focus when loaded

**Comment by Jefiozie** [comment]
okay agree, could you have a look at the tests?

**Comment by brianchance** [comment]
The test details are about the header and footer, not the autofocus. When I run the tests locally, I get success across the board. Will try to re-trigger the travis-ci

**Comment by brianchance** [comment]
I pulled the latest, not sure what the issue is, but it is the describe('Header & Footer' tests that are failing, not the autofocus tests.  

It seems to me that the header and footer templates were removed in the merge from you repo.

**Comment by Jefiozie** [comment]
Hi @brianchance  could you merge the master in your branch and see if the issue is fixed?
Thanks

**Comment by brianchance** [comment]
I have merged with the main branch. I am still unsure of what happened here. 

When someone (you?) merged into my branch - 3cca75a79e5f89a6011a0264b37e9f5be2405baf, it removed the header and footer directive but not the tests for the header and footer. My final commit removes those tests.

If 3cca75a79e5f89a6011a0264b37e9f5be2405baf was not supposed to be merged, I can work to get rid of it.

**Comment by nasvillanueva** [comment]
@brianchance Hi, is there any updates on this?

**Comment by brianchance** [comment]
@nasvillanueva Nope.

**Pull request closed without merging**